modules/odo-core-concepts.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-core-concepts_{context}"]
+
+= odo core concepts
+
+`odo` abstracts Kubernetes concepts into terminology that is familiar to developers:
+
+Application::
+A typical application, developed with a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native approach], that is used to perform a particular task.
++
+Examples of _applications_ include online video streaming, online shopping and hotel reservation systems.
+
+Component::
+A set of Kubernetes resources that can run and be deployed separately. A cloud-native application is a collection of small, independent, loosely coupled _components_. 
++
+Examples of _components_ include an API back-end, a web interface and a payment back-end.
+
+Project::
+A single unit containing your source code, tests, and libraries. Projects help achieve multi-tenancy, allowing multiple teams to run multiple applications at the same time, on a single cluster.
```

Comment:
I'm not sure about this. The first sentence describes the project on the developer machine, and the second one describe the Project Kubernetes resource (similar to Namespace).

Reply:
Agreed - will remove second sentence.